Packs/Coralogix/Integrations/Coralogix/Coralogix.yml

```
@@ -3,59 +3,103 @@ commonfields:
   id: Coralogix
   version: -1
 configuration:
-- defaultvalue: https://webapi.coralogix.com
+- additionalinfo: The Coralogix WebAPI endpoint for your account - Do not modify unless
+    instructed to by Coralogix Customer Success
+  defaultvalue: https://webapi.coralogix.com
   display: Coralogix WebAPI Endpoint URL
   name: webapi_url
   required: true
   type: 0
-- display: Private Key
+- additionalinfo: Your Coralogix account private key. Can be found in Coralogix under
+    Settings > Send your logs
+  display: Private Key
   name: privatekey
   required: true
   type: 0
```

```suggestion
  type: 4
```

Didn't we say we are changing this one to be encrypted?